custom_components/zwave_mqtt/entity.py

```diff
@@ -127,23 +127,23 @@ def value_changed(self, value):
     def device_info(self):
         """Return device information for the device registry."""
         return {
-            "identifiers": {(DOMAIN, self._value.node.node_id)},
-            "name": f"{self._value.node.node_manufacturer_name} {self._value.node.node_product_name}",
-            "manufacturer": self._value.node.node_manufacturer_name,
-            "model": self._value.node.node_product_name,
+            "identifiers": {(DOMAIN, self._value.primary.node.node_id)},
+            "name": f"{self._value.primary.node.node_manufacturer_name} {self._value.primary.node.node_product_name}",
+            "manufacturer": self._value.primary.node.node_manufacturer_name,
+            "model": self._value.primary.node.node_product_name,
         }
 
     @property
     def name(self):
         """Return the name of the entity."""
-        return f"{self._value.node.node_manufacturer_name} {self._value.node.node_product_name}: {self._value.label}"
+        return f"{self._value.primary.node.node_manufacturer_name} {self._value.primary.node.node_product_name}: {self._value.primary.label}"
 
     @property
     def state(self):
```

Comment:
We shouldn't override state. That's up to the abstract base class to define (light, switch etc)

Reply:
Only sensor is different, there we define state.